docs/03-pages/01-building-your-application/05-optimizing/10-testing.mdx

```diff
@@ -459,7 +459,7 @@ moduleNameMapper: {
 
 Add the Jest executable in watch mode to the `package.json` scripts:
 
-```jsx
+```json
```

Comment:
This is not valid JSON. You'll need to wrap in curly brackets

Reply:
Fixed this one and others.